code/modules/projectiles/guns/specialist.dm

```diff
@@ -420,6 +420,30 @@ Note that this means that snipers will have a slowdown of 3, due to the scope
 	SSmonitor.stats.miniguns_in_use -= src
 	return ..()
 
+// SG minigun
+
+/obj/item/weapon/gun/minigun/smart_microgun
+	name = "\improper T-85 LSWS 'Microgun'"
+	desc = "A four barreled Light Support Weapon Systems or a four-barreled 'Microgun' to those who prefer slang over technicality, this is a IFF-Capable light support weapon with a massive amount of firepower and thought put behind the construction.. Mind the windup, however. Uses 10x26 caseless"
```

Comment:
```suggestion
	desc = "A multi-barreled Light Support Weapon System or 'Microgun' to those who prefer slang over technicality. This is a IFF-capable light support weapon with a massive amount of firepower and thought put behind the construction... Mind the windup, however. It is chambered in 10x26 caseless."
```

Reply:
Shouldn't that "a" next to IFF-capable be "an"?